warehouse/query-core/src/main/java/datawave/query/QueryParameters.java

```diff
@@ -221,4 +223,9 @@ public class QueryParameters {
      * Used to specify model or DB fields that must be treated as strict (cannot be skipped if normalization fails)
      */
     public static final String STRICT_FIELDS = "strict.fields";
+
+    /**
+     * Controls whether a query's ID is logged on the tserver using {@link QueryLogIterator}
+     */
+    public static final String TSERVER_LOGGING_ACTIVE = "tserver.logging.active";
```

Comment:
Please move this constant to the QueryOptions class instead.  QueryParameters are for parameters specified by users when submitting queries.  This is purly used as an option on the query log iterator.

Reply:
Actually, now that I look at it you are setting this as a iterator option as well as pulling it out of the query parameters.  I don't believe the user submitting the query should be able to turn logging off and on.  That should be done when the query logic is being created (see QueryLogicFactory.xml).  You may want to add a line in there.  I think this constant should go away, and the default query planner should optionally setup the QueryLogIterator if that boolean is true in the configuration.